src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -82,8 +97,37 @@ def load_arguments(self, _):
         c.argument('webhook_name', completer=None)
 
     with self.argument_context('acr replication') as c:
-        c.argument('registry_name', options_list=['--registry', '-r'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
+        c.argument('registry_name', options_list=['--registry', '-r'])
         c.argument('replication_name', options_list=['--name', '-n'], help='The name of the replication.', completer=get_resource_name_completion_list(REPLICATION_RESOURCE_TYPE))
 
     with self.argument_context('acr replication create') as c:
         c.argument('replication_name', help='The name of the replication. Default to the location name.', completer=None)
+
+    with self.argument_context('acr build') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'])
+        c.positional('source_location', help="The local source code directory path (e.g., './src') or the URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') or a remote tarball (e.g., 'http://server/context.tar.gz').", completer=FilesCompleter())
+        c.argument('timeout', help='The build timeout in seconds.')
+
+    with self.argument_context('acr build-task') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'])
+        # build task parameters
+        c.argument('build_task_name', options_list=['--name', '-n'], help='The name of the build task.', completer=get_resource_name_completion_list(BUILD_TASK_RESOURCE_TYPE))
+        c.argument('alias', help='The alternative name for build task. Default to the build task name.')
+        c.argument('status', help='The current status of build task.', choices=['Enabled', 'Disabled'])
+        c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', choices=['Linux', 'Windows'])
+        c.argument('cpu', type=int, help='The CPU configuration in terms of number of cores required for the build.')
+        c.argument('timeout', type=int, help='Build timeout in seconds.')
+        c.argument('repository_url', options_list=['--context', '-c'], help="The full URL to the source code respository.")
+        c.argument('commit_trigger_enabled', help="Indicates whether the source control commit trigger is enabled.", choices=BOOL_CHOICES)
```

Comment:
For anywhere and everywhere "BOOL_CHOICES" is used, see:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags